### PR TITLE
Return az arns as well

### PR DESCRIPTION
**Pull request by rverma-nikiai**
Nice to have.

### CODE REVIEW

**Comment by aknysh** [comment]
@rverma-nikiai  can you please rebuild the `README.md` again by running the following commands:
```
make init
make readme/deps
make readme
```

Before, you updated README and then merged your changes, so the new output did not end up in README

**Comment by rverma-nikiai** [comment]
@aknysh already did using 
```
make init
make readme/deps
make readme
```

**Comment by rverma-nikiai** [comment]
@aknysh done

File: outputs.tf [hunk 1]
```diff
@@ -12,3 +12,8 @@ output "az_ngw_ids" {
   value       = "${zipmap(var.availability_zones, coalescelist(aws_nat_gateway.public.*.id, local.dummy_az_ngw_ids))}"
   description = "Map of AZ names to NAT Gateway IDs (only for public subnets)"
 }
+
+output "az_subnet_arns" {
+  value       = "${zipmap(var.availability_zones, coalescelist(aws_subnet.private.*.arn, aws_subnet.public.*.arn))}"
+  description = "Map of AZ names to subnet arns"
```
Comment (aknysh) [inline outputs.tf]:
```suggestion
  description = "Map of AZ names to subnet ARNs"
```

**Comment by osterman** [comment]
@rverma-nikiai bump

**Comment by aknysh** [comment]
@rverma-nikiai  let's finish this PR :)

**Pull request merged**